graph-builder/src/graph.rs

```diff
@@ -169,7 +164,7 @@ impl HasRegistry for State {
 }
 
 #[allow(clippy::useless_let_if_seq)]
-pub fn run<'a>(settings: &'a config::AppSettings, state: &State) -> ! {
+pub async fn run<'a>(settings: &'a config::AppSettings, state: &State) -> ! {
```

Comment:
Whoa!

Reply:
It doesn't magically give us concurrency of what's going on inside though, we'll still have to work for that :wink: